core/controllers/payload_validator.py

```diff
@@ -21,13 +21,16 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
+import python_utils
 import schema_utils
 
 from typing import Any, Dict, List, Text, Tuple # isort:skip  pylint: disable= wrong-import-order, wrong-import-position, unused-import, import-only-modules
 
 
-def validate(handler_args, handler_args_schemas, allowed_extra_args):
-    # type: (Any, Any, bool) -> Tuple[Dict[Any, Any], List[Text]]
+def validate(
+        handler_args, handler_args_schemas, allowed_extra_args,
+        allow_string_to_bool_conversion):
+    # type: (Any, Any, bool, bool) -> Tuple[Dict[Any, Any], List[Text]]
```

Comment:
Here and elsewhere, try to use Any as little as possible. Keys of dicts are not Any and each argument should not be Any to start with. Please talk to @hardikkat24 and @sajalasati if you need guidance on this.

Reply:
Updated for output dict.
Regarding input,  ```Any``` are present because the dict's (handler_args, handler_args_schemas) are empty in some case -- when a handler class do not have any param.
```Dict[Text, Any]``` will fail for those conditions, thus I have written ```Any``` for first two arguments.
@hardikkat24 what is your opinion regarding this?
Thank you.